toyapp/test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+class UsersControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @user = users(:one)
+  end
+
+  test "should get index" do
```

Comment:
tido on test description

Reply:
Awesome will make sure  I use the correct convention

Reply:
Awesome will make sure  I use the correct convention